web_external/views/body/DataPanel.js

```diff
@@ -236,7 +238,7 @@ export default Panel.extend({
 
     deleteDatasetEvent: function (event) {
         if ($(event.currentTarget).hasClass('icon-disabled')) {
```

Comment:
We should probably invert this conditional so we don't have an empty block and then an else.